### DISCUSSION TITLE
Support view decoded query string parameters

### DISCUSSION THREAD

**Discussion by dengyakui** [datalust/seq-tickets#1452] [Ideas]
If endpoint’s url contains some encoded query string,  for example: 
- ‘/api/weather?country=%20%2faaa’
- ‘/api/weather?country= %20%2fbb’

Seq show it’s encoded string values, it’s hard to see the real value, developers  want to see the decoded values like chrome dev tool does. ( view raw / view decoded)

By the way,  when we do some aggregation , by the request path, 
it’s hard to trim the query string parameters to to do group by.

Hope Seq to treat the  RequestPath like  ‘/api/weather?a=xxx’  to ‘/api/weather’, 
like a BaseURL, trim the query string. for better aggregation

**Comment by nblumhardt** [comment]
Thanks for the suggestion! This overlaps substantially with #1386 and we'd likely consider these (URI parsing and encoding) together 👍

**Comment by nblumhardt** [comment]
Thanks again for the suggestion, @dengyakui. We don't expect to tackle this in 2022.1 or .2, so we'll move this issue to a discussion to collect further feedback.
